sjb/config/test_cases/pull-ci-openshift-cluster-api-actuator-pkg-master-e2e.yml

```diff
@@ -49,6 +50,13 @@ extensions:
           xargs curl -O -L
         chmod u+x kustomize_*_linux_amd64
         sudo mv kustomize_*_linux_amd64 /usr/bin/kustomize
+    - type: "script"
+      title: "Build cluster autoscaler"
+      script: |-
+        export GOPATH=/data
+        cd $GOPATH/src/github.com/openshift/kubernetes-autoscaler
+
+        sudo imagebuilder -f images/cluster-autoscaler/Dockerfile -t "docker.io/openshift/origin-cluster-autoscaler:v4.0" .
```

Comment:
Looks good based on https://github.com/openshift/cluster-autoscaler-operator/blob/master/install/04_deployment.yaml#L44

Reply:
`imagebuilder` is installed later in the process. Please, put the change after `Deploy cluster autoscaler operator` section.

Reply:
Added a new step to install imagebuilder.